### PR TITLE
feat(COR-1772): Fix no patient numbers shown on zkh opnames GM level

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard29**
Extra property to fix inaccurate data for in get-data.ts

### CODE REVIEW

File: packages/app/src/static-props/get-data.ts [hunk 1]
```diff
@@ -312,13 +312,15 @@ export function createGetArchivedChoroplethData<T1, T2>(settings?: {
 function replaceInaccurateLastValue(data: any) {
   const metricsInaccurateItems = ['intensive_care_nice', 'hospital_nice'];
 
-  const inaccurateMetricProperty = 'admissions_on_date_of_admission_moving_average_rounded';
+  const inaccurateMetricProperties = ['admissions_on_date_of_admission_moving_average_rounded', 'admissions_in_the_last_7_days'];
 
   const metricsWithInaccurateData = metricsInaccurateItems.filter((m) => m in data) as (keyof typeof data & keyof typeof metricsInaccurateItems)[];
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/static-props/get-data.ts:317]: Shorthands for arguments like `x, `i`, `m` etc. reduce readability. I recommend refactoring a little bit to get rid of the shorthand.

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
Please see my comment, looks good otherwise.

File: packages/app/src/static-props/get-data.ts (hunk 1 above)
Comment (VWSCoronaDashboard30) [reply packages/app/src/static-props/get-data.ts:317]: You mean having `metric` instead of `m`?

Comment (VWSCoronaDashboard28) [reply packages/app/src/static-props/get-data.ts:317]: Yes

**Review by VWSCoronaDashboard30 — APPROVED** [review]

File: packages/app/src/static-props/get-data.ts (hunk 1 above)
Comment (VWSCoronaDashboard29) [reply packages/app/src/static-props/get-data.ts:317]: As the `m` notation already existed before we made this change I think we'l leave it for now. Let's take note of this in the future.

**Pull request merged**